### DISCUSSION TITLE
does the driver generate warnings and alarms?

### DISCUSSION THREAD

**Discussion by lennycb** [Louisvdw/dbus-serialbattery#336] [Q&A] [answered]
I recently got alarms displayed in Venus.
![image](https://user-images.githubusercontent.com/23618370/206849155-99034e32-5e8e-44be-9313-0fbcd2db04de.png)
Now I'm not sure who set off those alarms.
![image](https://user-images.githubusercontent.com/23618370/206849171-5395a5d9-1b2d-4bf5-b7e2-4f46a94220a1.png)
SerialBattery(Daly) shows nothing, the Daly app says there was a SoCLowLV2.

I actually thought that I had rounded up the tensions in the ESS assistant to such an extent that no more alarms came from there either.

hmm Do you have an idea? Is that from the driver?

**Comment by Louisvdw** [comment, marked as answer]
The driver does publish alarms to the GX system, but in this case the alarm came from your inverter. 
The first row show the deivce where the alarm is from. In this case the Multiplus II. These settings from the alarms in the Multiplus will come from what you set using the VE.Configure application. Some settings can also be done in VictronConnect now as well.
